projects/vdk-plugins/vdk-jobs-troubleshooting/src/vdk/plugin/jobs_troubleshoot/jobs_troubleshoot_plugin.py

```diff
@@ -5,21 +5,59 @@
 """
 import logging
 from typing import List
+from typing import Optional
 
 from vdk.api.plugin.hook_markers import hookimpl
 from vdk.api.plugin.plugin_registry import IPluginRegistry
+from vdk.internal.builtin_plugins.run.job_context import JobContext
 from vdk.internal.core.config import ConfigurationBuilder
+from vdk.plugin.jobs_troubleshoot.api.troubleshoot_utility import ITroubleshootUtility
 from vdk.plugin.jobs_troubleshoot.troubleshoot_configuration import add_definitions
+from vdk.plugin.jobs_troubleshoot.troubleshoot_utilities.utilities_registry import (
+    get_utilities_to_use,
+)
 
 log = logging.getLogger(__name__)
 
 
 class JobTroubleshootingPlugin:
+    def __init__(self):
+        self.troubleshooting_utils: Optional[List[ITroubleshootUtility]] = []
```

Comment:
Please take into account https://github.com/vmware/versatile-data-kit/wiki/Coding-Standard#clarification-on-access-modifiers-in-python